### PR TITLE
Favicons and translations boilerplate

### PR DESCRIPTION
**Pull request by jmichaliga**
Some in the Copy-and-translate team need more boilerplate.
Added Favicons while I was there.

### CODE REVIEW

File: public/locales/stake.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (GarrettJMU) [inline public/locales/stake.json:1]: should we add in other languages here?

File: src/components/Stake/index.tsx [hunk 2]
```diff
@@ -92,19 +97,25 @@ export const StakeView: FC<StakeUnstakeBoxProps> = ({ stakingMode }) => {
     }
   }
 
-  const determineText = () => {
+  const determineText = (
+    connectText = translate('connectTo') || 'Connect to ETH mainnet',
+    approvingText = translate('stake') || 'Approving...',
+    sendingText = translate('stake') || 'Sending...'
```
Comment (GarrettJMU) [inline src/components/Stake/index.tsx:103]: Instead of prop drilling we should do this at the component level

File: src/pages/stake.tsx [hunk 3]
```diff
@@ -35,15 +43,25 @@ export default function Deposit() {
...
 
-      <DepositDetails stakingMode={stakingMode} />
+      <DepositDetails
+        stakingMode={stakingMode}
+        cta={translate('cta') || 'Deposit, yield, support'}
+        withdrawCta={
+          translate('withdrawCta') || 'Withdraw all deposited assets'
+        }
+        stakedValueLabel={translate('stakedValueLabel') || 'Your Staked Value'}
+        totalStakedLabel={translate('totalStakedLabel') || 'Total Staked'}
```
Comment (GarrettJMU) [inline src/pages/stake.tsx:63]: let's move to the component level

File: src/pages/stake.tsx [hunk 4]
```diff
@@ -19,7 +23,11 @@ export default function Deposit() {
   return (
     <Flex direction="column" gap={16} width="100%" align="center">
       <Flex direction="column" align="center" justify="center">
-        <RedeemSwitch onChange={tabChanged} />
+        <RedeemSwitch
+          stake={translate('stake') || 'Stake'}
+          unstake={translate('unstake') || 'Unstake'}
```
Comment (GarrettJMU) [inline src/pages/stake.tsx:28]: let's move to base component

**Review by GarrettJMU — CHANGES_REQUESTED** [review]

File: public/locales/stake.json (hunk 1 above)
Comment (jmichaliga) [reply public/locales/stake.json:1]: yeah, this is here to provide some boilerplate and send to the copy team to unblock translators.

**Pull request merged**